docs/contributors/document.md

```diff
@@ -35,41 +35,49 @@ To update an existing page:
 2. Create a branch to work, for example `docs/update-contrib-guide`.
 3. Make the necessary changes to the existing document.
 4. Commit your changes.
-5. Create a pull request with "Documentation" label.
+5. Create a pull request using "\[Type\] Documentation" label.
 
 ### Create a New Document
 
-To add a new documentation page:
+To add a new documentation page requires a working JavaScript development environment to build the documentation, see the [JavaScript build setup documentation](/docs/designers-developer/developers/tutorials/javascript/js-build-setup.md):
 
-1. Create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder.
-2. Add item to the [toc.json](https://github.com/WordPress/gutenberg/blob/master/docs/toc.json) hierarchy.
-3. Update `manifest.json` by running `npm run docs:build`.
-4. Commit `manifest.json` with other files updated.
+1. Create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder, use lower-case, no spaces, if needed a dash separator.
```

Comment:
This might be obvious to others but I would make it clear you need to add the .md at the end of the file. This made sense in retrospect but for someone as new as me this escaped me.

Reply:
Good suggestion, the more clear we can be the better. I updated and added the extension too.